test/language/expressions/generators/params-trailing-comma-length.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2016 Jeff Morrison. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+description: Check that trailing commas are permitted in function argument lists
```

Comment:
Sure, it doesn't have to be as detailed as Mike's tests for example, but something like https://github.com/tc39/test262/pull/735 is a good example.

In this particular case documenting the type of function and/or why the test is interesting (e.g. rest) would be helpful.